# Sent by a procman deputy when one or ore child processes write something to
# stdout/stderr.

# Output from multiple processes can be contained in a single message. This
# can be done to reduce the number of individual packets transmitted by the
# deputy.

# Also sent when the deputy itself has something to say.

time timestamp

# Which deputy sent this message.
string deputy_id

# size of the arrays of commands/text
int16 num_commands

# Id of each child process.  Empty string if the message is generated by
# the deputy
string[] command_ids

string[] text

